pkg/util/util_test.go

```diff
@@ -55,3 +55,25 @@ func TestThroughputCalc(t *testing.T) {
 		}
 	}
 }
+
+func TestMaskPassword(t *testing.T) {
+	testData := map[string]string{
+		"password='foobar' host='localhost'":   "password='****' host='localhost'",
```

Comment:
Maybe add an example where password is not at the start of the string

Reply:
Thanks for that, tweaked the regex a bit, take a look.